editor/src/components/canvas/canvas-strategies/strategies/test-static-reparent-strategy.tsx

```diff
@@ -1,23 +0,0 @@
-import {
```

Comment:
I think this file can be deleted (it is not used in canvas-strategies anymore)

Reply:
fixed in https://github.com/concrete-utopia/utopia/pull/3798/commits/0db863999a814ca1e76a95bcd52206264c386256

Reply:
it is deleted :D

Reply:
🤦